### PR TITLE
Safety Moth(TM) posters from your friends at Nanotrasen! (Now with artist approval)

### PR DESCRIPTION
**Pull request by AyyRobotics**
## About The Pull Request

Adds 7 Safety Moth(TM) posters, and one decommissioned Safety Pill(TM?) poster.
Themoffs
Themoffs2

Now with artist permission!
image

## Why It's Good For The Game

1. Giving the crew beneficial posters that promote safe workplace procedures.
2. Safety Moth.
3. More posters in general.

## Changelog
:cl:
add: Added Safety Moth(TM), and Safety Pill(TM?) posters.
imageadd: Updated contraband.dmi to add new poster icons.
/:cl:

### CODE REVIEW

**Comment by remove32** [comment]
how could you

**Comment by AyyRobotics** [comment]
> how could you

How could I _promote safety?_

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -606,4 +606,44 @@
...
+	icon_state = "poster36_legit"
+
+/obj/structure/sign/poster/official/moth2
+	name = "Safety Moth - BoH"
+	desc = "This informational poster uses Safety Moth(TM) to inform the viewer of the dangers of Bags of Holding. It's signed by 'AspEv'."
+	icon_state = "poster37_legit"
+
+/obj/structure/sign/poster/official/moth3
+	name = "Safety Moth - Hardhats"
+	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to wear hardhats in cautious areas. It's like a lamp for your head! It's signed by 'Aspev'."
```
Comment (ike709) [inline code/game/objects/effects/contraband.dm]:
```suggestion
	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to wear hardhats in cautious areas. It's like a lamp for your head! It's signed by 'AspEv'."
```

File: code/game/objects/effects/contraband.dm [hunk 2]
```diff
@@ -606,4 +606,44 @@
...
+	icon_state = "poster41_legit"
+
+/obj/structure/sign/poster/official/moth7
+	name = "Safety Moth - Methamphetamine"
+	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to seek CMO approval before cooking methamphetamine. You shouldn't even be making this. It's signed by 'AspEv'."
+	icon_state = "poster42_legit"
+
+/obj/structure/sign/poster/contraband/pill1
+	name = "Safety Pill - Methamphetamine"
+	desc = "A decommisioned poster that uses Safety Pill(TM?) to promote less-than-legal chemicals. This is one of the reasons we stopped outsourcing these posters. It's partially signed by 'Aspev'."
```
Comment (ike709) [inline code/game/objects/effects/contraband.dm]:
```suggestion
	desc = "A decommisioned poster that uses Safety Pill(TM?) to promote less-than-legal chemicals. This is one of the reasons we stopped outsourcing these posters. It's partially signed by 'AspEv'."
```

**Comment by Crossedfall** [comment]
I'll reach out to you via Discord on the exact specifics I need to ensure that we're fully covered here.

**Comment by BettonnCZ** [comment]
yes, add it. You have to!

**Review by Crossedfall — APPROVED** [review]
I have been in contact with the original artist. They have given BeeStation explicit permission to use their character and posters within our codebase. 

I've added the artist in as a contributor for this PR so they'll also be attributed directly in our codebase.

**Pull request merged**